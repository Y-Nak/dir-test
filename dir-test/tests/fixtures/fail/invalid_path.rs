use dir_test::dir_test;

#[dir_test(
    dir: "../foo/"
)]
fn foo(fixture: Fixture<&str>) {}

#[dir_test(
    dir: "/__NONE__"
)]
fn foo(fixture: Fixture<&str>) {}

fn main() {}
