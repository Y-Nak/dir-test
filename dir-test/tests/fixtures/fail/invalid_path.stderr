error: `../foo` is not an absolute path
 --> tests/fixtures/fail/invalid_path.rs:4:10
  |
4 |     dir: "../foo/"
  |          ^^^^^^^^^

error: `/__NONE__` does not exist
 --> tests/fixtures/fail/invalid_path.rs:9:10
  |
9 |     dir: "/__NONE__"
  |          ^^^^^^^^^^^
